tests/resources/test_predictor.py

```diff
@@ -79,46 +79,40 @@ def test_archive(valid_label_fractions_predictor_data):
     predictors_path = PredictorCollection._path_template.format(project_id=pc.project_id)
     pred_id = valid_label_fractions_predictor_data["id"]
 
-    session.set_responses(None, valid_label_fractions_predictor_data, paging_response(valid_label_fractions_predictor_data))
+    session.set_responses(None, valid_label_fractions_predictor_data)
 
     with pytest.deprecated_call():
         archived_predictor = pc.archive(pred_id)
 
     assert session.calls == [
         FakeCall(method='PUT', path=f"{predictors_path}/{pred_id}/archive", json={}),
-        FakeCall(method='GET', path=f"{predictors_path}/{pred_id}"),
-        FakeCall(method='GET', path=predictors_path, params={"filter": "archived eq 'true'", 'per_page': 20})
+        FakeCall(method='GET', path=f"{predictors_path}/{pred_id}")
     ]
 
-    assert archived_predictor.is_archived
```

Comment:
Was removing this intentional?

Reply:
Yes. Currently, `archived_predictor` is just the payload we told it to respond with on line 82. Previously, the value returned by `is_archived` would be impacted by processing on the response of a subsequent call, in order to present a value the user expected before they were exposed to versions.